mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule ICalendar.Mixfile do
   use Mix.Project
 
-  @version "0.8.1"
+  @version "0.8.2"
```

Comment:
Bump this to `1.0.0` please :)

Reply:
https://github.com/lpil/icalendar/pull/36/commits/dbc5c4dee7500a8ec3321f11e77478ccd6817abf done :)